protocol/invalid-block-progression.md

```diff
@@ -0,0 +1,60 @@
+# Purpose
+
+The purpose of this document is to progress a design originally proposed by @protolambda and contributed to
+by Hamdi [here](https://github.com/ethereum-optimism/specs/pull/88). The goal is to reduce resource usage
+by the derivation pipeline when invalid blocks are proposed. This is helpful for ensuring that the fault
+proof program cannot be denial of service attacked as well as reducing the latency of reorgs, which is
+especially important in the context of interop.
+
+# Summary
+
+New derivation pipeline invariants are introduced that are meant to reduce large resets of the derivation pipeline
+when handling invalid batches.
```

Comment:
It would be nice to more explicitly explain (probably below, not here) why the new rules avoid larger resets. This could include explaining what the _current_ behavior is under the preconditions of the new rules.